win32.c

```diff
@@ -21,6 +21,26 @@ typedef UINT32 *PQOS_FLOWID;
 #endif
 
 static enet_uint32 timeBase = 0;
+
+#if defined(WINAPI_FAMILY) && WINAPI_FAMILY == WINAPI_FAMILY_APP
+# define ON_UWP
+#endif
+
+#ifdef ON_UWP
+void timeBeginPeriod(int t) {
+
+}
+void timeEndPeriod(int t) {
+
+}
+
+enet_uint32 timeGetTime()
+{
+    return GetTickCount64();
```

Comment:
I pushed a change to remove usage of these `time*` functions, so you can drop this part.